service-jehlomat/src/main/resources/files/swagger.yaml

```diff
@@ -583,3 +596,15 @@ definitions:
       organizationId:
         type: integer
         example: 0
+  LocationTriple:
+    type: object
+    properties:
+      id:
+        type: string
+        example: CZ0323
+      name:
+        type: string
+        example: Praha
+      type:
+        type: string
```

Comment:
I would suggest to describe the type property as an enumeration with all possible values to make FE life easier.

Reply:
Hopefully I understand your comment correctly: 57eacf41131b1053ca9146d369b326ba98010005 :-)

Reply:
4ff00768e23dc94f3abe49abe4c17eb3b2213536